### PR TITLE
3950 + fixed css names

### PR DESCRIPTION
**Pull request by oss-devbranch**
Slavik do it

### CODE REVIEW

File: web/themes/uno/css/blocks.scss [hunk 1]
```diff
@@ -335,4 +335,30 @@ footer.site-footer {
...
+
+      a:link, a:visited {
+        color: #000;
+        font-weight: bold;
+        &:hover{
+          text-decoration: underline;
+        }
+      }
+
+      div:nth-child(2) {
```
Comment (henzzapp) [inline web/themes/uno/css/blocks.scss:354]: @oss-devbranch 
not a safe selector. It'll target all child nested divs with 2nd position.

maybe use:
`&  > div:nth-child(2)` ?

File: web/themes/uno/css/blocks.scss [hunk 2]
```diff
@@ -335,4 +335,30 @@ footer.site-footer {
...
+        display: block;
+        padding: 5px 0;
```
Comment (henzzapp) [inline web/themes/uno/css/blocks.scss:356]: I see you use 5px and 10px values for spacing.

How about creating a style variable `spacingUnit: 5px`
And use an SCSS function e.g `spacing(1)` to get 5px (and `spacing(2)` to get 2* spacingUnit) to get 10px?

Apply this if you plan to use it as spacing multiplier further onwards

File: web/themes/uno/css/blocks.scss (hunk 1 above)
Comment (oss-devbranch) [reply web/themes/uno/css/blocks.scss]: +

File: web/themes/uno/css/blocks.scss (hunk 2 above)
Comment (oss-devbranch) [reply web/themes/uno/css/blocks.scss]: fixed, i update branch

**Pull request merged**